### PR TITLE
Changed Fonts and Margin +  Minor changes in Footer + Query Form Added

### PR DESCRIPTION
**Pull request by suyashagno3**
1. Changed Fonts and colors as decided
2. Improved Padding and Margin
3.  Minor changes in Footer as mentioned in the design
4. Added Query Form (responsive)
Query form
Responsive
responsive query form

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -214,6 +216,101 @@
...
+  align-items: center;
+}
+.form-container {
+  margin: 100px;
+  font-family: 'Roboto', sans-serif;
+  font-style: italic;
+  font-weight: 600;
+}
+
+@media screen and (max-width: 600px) {
```
Comment (Anshultilondiya) [inline frontend/src/App.css:285]: Use Material-Ui Grid or Form for making it responsive

**Review by Anshultilondiya — CHANGES_REQUESTED** [review]

**Comment by Anshultilondiya** [comment]
@suyashagno3 could you please resolve conflicts in App.css

**Comment by suyashagno3** [comment]
@Anshultilondiya  Done

**Comment by suyashagno3** [comment]
@Anshultilondiya Fixed it...

**Pull request merged**